pom.xml

```diff
@@ -1859,6 +1866,18 @@
                 <version>42.6.0</version>
             </dependency>
 
+            <dependency>
```

Comment:
Can we move to these Iceberg nessie module to trino-iceberg?

Reply:
I think it is declaring the dependencies required for trino-iceberg under `dependencyManagement` and actual dependency is only present in trino-iceberg.

Are you suggesting no need to follow that style?

Reply:
There's no need to define the dependency in root pom.xml unless it's used from several modules.

Reply:
Handled for newly added code from this PR as per your suggestion